VMS/Shared/LoginDisplay.razor

```diff
@@ -15,20 +15,71 @@
                     <ul class="dropdown-menu user-section__list list-unstyled bg-white">
                         @if (context.User.IsInRole(Role.User.ToString()))
                         {
-                            <li class="text-center">
-                                <a href="@Routes.User" class="px-3 pt-3 pb-2 text-decoration-none fw-bold user-section__list__profile">Trang cá nhân</a>
+                            <li class="user-info">
+                                <a href="@Routes.User" class="d-flex px-3 pt-4 text-decoration-none align-items-center">
+                                    <img src="./img/HamburgerMenu/AvaUser.png" class="user-img" />
+                                    <div class="d-flex flex-column text-start ps-3">
+                                        @foreach (var item in fakeDataUsers)
+                                        {
+                                            <h3 class="py-1 mb-0">@item.Name</h3>
+                                            <h4 class="py-1 mb-0">@item.MemberOf</h4>
+                                        }
+                                    </div>
+                                </a>
                             </li>
```

Comment:
I see that from line 18-29 is the same as line 50-61. So please move it outside the if-else to reuse for both roles.